contributor_docs/hi/contributor_guidelines.md

```diff
@@ -0,0 +1,528 @@
+# योगदानकर्ता दिशानिर्देश
+
+योगदानकर्ता दिशानिर्देशों में आपका स्वागत है! यह दस्तावेज़ उन नए योगदानकर्ताओं के लिए है जो p5.js में कोड का योगदान करना चाहते हैं, ऐसे योगदानकर्ता जो कुछ तकनीकी चरणों पर अपनी यादें ताज़ा करना चाहते हैं, या p5.js में कोड योगदान के साथ कुछ और करना चाहते हैं।
+
+यदि आप p5.js रिपॉजिटरी के बाहर योगदान करना चाहते हैं (ट्यूटोरियल लिखना, कक्षाओं की योजना बनाना, कार्यक्रमों का आयोजन करना), तो कृपया इसके बजाय अन्य प्रासंगिक पृष्ठों पर एक नज़र डालें। प्रबंधकों या अनुरक्षकों को समस्याएँ की समीक्षा करने और अनुरोधों को खींचने के संबंध में [कार्यपालिका दिशानिर्देश](https://github.com/processing/p5.js/blob/main/contributor_docs/steward_guidelines.md) अधिक उपयोगी लग सकते हैं।
```

Comment:
hello, this whole text looks too typical can we have some alternatives here?

Reply:
done